google-cloud-automl/CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 # Release History
 
+### 0.2.0 / 2019-08-14
+
+#### Features
+
+* Deprecate ColumnSpec#top_correlated_columns
```

Comment:
I think the following edit makes more sense for the changelog, since the Autosynth PR contained several different changes. I personally would list added features before deprecated features. And we only need to list "Updated documentation" once.

```
* Add BatchPredictResult#metadata.
* Add ConfusionMatrix#display_name
* Add TableSpec#valid_row_count
* Deprecate ColumnSpec#top_correlated_columns
* Update documentation
```

Reply:
I wonder if changes like this would be a good candidate for the "Edit file" feature on GitHub's UI.

Reply:
If we have sub-headings, would it make more sense for a deprecation to go under something other than `Features`?

Reply:
we could consider introducing a `deprecate:` prefix?

Reply:
The problem is that Autosynth changes are not always just about one thing. So we describe all the different types of changes in the conventional commit body. That commit has both features and deprecations. This is partially why I want to make sure we can always update the changelog to correct things like this.

Reply:
I don't know how comfortable you are with TypeScript, *but* you could gradually add more of these rules here:

https://github.com/googleapis/release-please/blob/master/src/releasers/ruby-yoshi.ts#L166

See how I'm already pushing in a generated commit to represent the fact that docs have been added? You could extend on this logic further to look for some of your additional shorthands.